bigquery/google/cloud/bigquery/schema.py

```diff
@@ -43,10 +43,16 @@ def __init__(self, name, field_type, mode='NULLABLE', description=None,
         self.description = description
         self.fields = fields
 
-    def __eq__(self, other):
+    def __key(self):
```

Comment:
Sure, I'll add one. Do you generally have docstrings for private methods (starting with `_`)?